.idea/modules.xml

```diff
@@ -4,7 +4,7 @@
     <modules>
```

Comment:
I think we should provide more appropriate gitignore file

Reply:
Done. Also deleted all files under .idea folder. [commit](https://github.com/Trendyol/android-ui-components/pull/1/commits/3300580e4dd4fdecba5fa70af7e1f09c3777cccd)